src/util/postgres_client.js

```diff
@@ -478,6 +479,87 @@ class BulkOp {
     }
 }
 
+async function multi_collection_bulk(bulk_per_collection) {
+
+    const client = PostgresClient.instance();
+
+    const queries = _.flatten(Object.values(bulk_per_collection).map(bulk => bulk.get_queries()));
+    const transaction = new PgTransaction(client);
+
+    let ok = false;
```

Comment:
this is the same code as execute() of bulkOp class, I don't see a value in maintaining this code in 2 different functions, easily can move to a helper function

Reply:
Makes sense. Done.